Makefile

```diff
@@ -151,5 +151,8 @@ compare-flobject:
 	@python .ci/compare_flobject.py
 
 cleanup-previous-docker-containers:
-	@if [ -n "$(docker ps -a -q)" ]; then docker stop $(docker ps -a -q); fi
+	@if [ -n "$(docker ps -a -q)" ]; then \
+		docker inspect --format='{{.Config.Labels.test_name}}' $(docker ps -a -q); \
```

Comment:
Prints unittest that leaves stale containers

Reply:
This will help to identify tests which require explicit `exit()` call as mentioned in above comment.